docs/reference/client-side-encryption.txt

```diff
@@ -324,6 +324,203 @@ master key and create data keys, see the following sections of this tutorial:
 - `Creating A Master Key`_,
 - `Creating A Data Key`_,
 
+Queryable Encryption
+====================
+Queryable encryption is a new feature in MongoDB 6.0. It also requires
+libmongocrypt version 1.5.0-rc1 or above.
+
+Usage of automatic queryable encryption is very similar to previously
+known client-side Encryption. Below is an example of automatic queryable
```

Comment:
Can this say what the differences are? Is it just using `:encrypted_fields` instead of `:schema_map`?

Also Encryption is capitalized while nothing else is.

Reply:
Apparently, some server docs were published, so I added a link.